Realm/Realm.Unity/Editor/Extensions/PropertyDefinitionExtensions.cs

```diff
@@ -154,6 +154,11 @@ internal static bool IsString(this PropertyDefinition property)
         return property.PropertyType.FullName == StringTypeName;
     }
 
+    internal static bool IsMixed(this PropertyDefinition property)
```

Comment:
[nit] Let's rename to `IsRealmValue` - since that's the official name in the SDK, it will likely be a bit confusing for someone new reading the code.